### ISSUE TITLE
Where can I get the test set Set8?

### ISSUE THREAD

**Issue by Ssakura-go** [JingyunLiang/RVRT#12]
Hi, dear author,
I'd like to train an RVRT model on my dataset. Now I try to train successfully on your training dataset, however, there also requires a test set called Set8 which is unavailable by your shared link. 
Could you please share the available link with me? Hope for your quick prompt reply! Thank you very much~

**Comment by Ssakura-go** [comment]
> https://github.com/JingyunLiang/VRT/releases/download/v0.0/testset_Set8.tar.gz

Thanks for your sharing!
